engine/apps/slack/scenarios/alertgroup_appearance.py

```diff
@@ -13,6 +13,7 @@
 from .step_mixins import CheckAlertIsUnarchivedMixin, IncidentActionsAccessControlMixin
 
 
+# TODO:
 class OpenAlertAppearanceDialogStep(
     CheckAlertIsUnarchivedMixin, IncidentActionsAccessControlMixin, scenario_step.ScenarioStep
 ):
```

Comment:
still need to refactor these `scenario_step.ScenarioStep` subclasses as they define an `ALLOWED_ROLES` property. Could use some help on this.